wherehows-web/mirage/fixtures/owners.ts

```diff
@@ -14,7 +14,7 @@ export default <Array<IOwner>>[
     namespace: OwnerUrnNamespace.corpUser,
     source: OwnerSource.Ui,
     subType: null,
-    type: 'Owner'
+    type: 'DataOwner'
```

Comment:
For the mirage fixtures, let's used the OwnerType `enum` so we don't have to change it here in the future.

`type: OwnerType.Owner`